packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,34 @@
+{
+  "pci_database_add_title": "Créer un serveur de données",
+  "pci_database_add_description1": "Créer une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, merci de",
+  "pci_database_add_description2": "créer un réseau privé",
+  "pci_database_add_description3": "ou un",
+  "pci_database_add_description4": "sous-réseau",
+  "pci_database_add_description5": "d’abord.",
```

Comment:
This is going to be hard to translate to something meaningful in other languages I think

Reply:
Yeah... We had similar requirements earlier too and we don't have a proper solution to deal with this, as yet